ci/rebuild-docker-images.py

```diff
@@ -485,6 +488,98 @@ def playwright_build(distro, arch):
+    LC_ALL=C.UTF-8 \
+    DISPLAY=:0.0 \
+    DISPLAY_WIDTH=1920 \
+    DISPLAY_HEIGHT=900
+
+
+WORKDIR /
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update
+RUN apt install -y ca-certificates curl git cpu-checker supervisor vim bash wget unzip xvfb x11vnc fluxbox xterm novnc net-tools htop libpulse-dev libnss3 libxcursor1 libasound2 libqt5gui5 libc++-dev libxcb-cursor0 htop tree tar gzip
+
+
+ARG ARCH="x86_64"
+ARG TARGET="google_apis_playstore"
+ARG API_LEVEL="{distro[1]}"
+ARG BUILD_TOOLS="{distro[1]}.0.0"
+ARG ANDROID_API_LEVEL="android-$API_LEVEL"
+ARG ANDROID_APIS="$TARGET;$ARCH"
+ARG EMULATOR_PACKAGE="system-images;$ANDROID_API_LEVEL;$ANDROID_APIS"
+ARG PLATFORM_VERSION="platforms;$ANDROID_API_LEVEL"
+ARG BUILD_TOOL="build-tools;$BUILD_TOOLS"
+ARG ANDROID_CMD="commandlinetools-linux-11076708_latest.zip"
+ARG ANDROID_SDK_PACKAGES="$EMULATOR_PACKAGE $PLATFORM_VERSION $BUILD_TOOL platform-tools"
+
+ENV ANDROID_SDK_ROOT=/opt/android
+ENV PATH "$PATH:$ANDROID_SDK_ROOT/cmdline-tools/tools:$ANDROID_SDK_ROOT/cmdline-tools/tools/bin:$ANDROID_SDK_ROOT/emulator:$ANDROID_SDK_ROOT/tools/bin:$ANDROID_SDK_ROOT/platform-tools:$ANDROID_SDK_ROOT/build-tools/$BUILD_TOOLS"
+ENV DOCKER="true"
+
+#============================================
+# Install required Android CMD-line tools
+#============================================
+RUN wget https://dl.google.com/android/repository/$ANDROID_CMD -P /tmp && \
+              unzip -d $ANDROID_SDK_ROOT /tmp/$ANDROID_CMD && \
+              mkdir -p $ANDROID_SDK_ROOT/cmdline-tools/tools && cd $ANDROID_SDK_ROOT/cmdline-tools &&  mv NOTICE.txt source.properties bin lib tools/  && \
+              cd $ANDROID_SDK_ROOT/cmdline-tools/tools && ls
+
+#============================================
+# Install required package using SDK manager
+#============================================
+RUN yes Y | sdkmanager --licenses
+RUN yes Y | sdkmanager --verbose --no_https $ANDROID_SDK_PACKAGES
+
+#============================================
+# Create required emulators
+#============================================
+
+ARG EMULATOR_NAME="emulator1"
+ARG EMULATOR_DEVICE="pixel_6" # all emulators are created with the pixel 6 spec for now
+
+RUN yes | sdkmanager emulator
+RUN echo "no" | avdmanager --verbose create avd --force --name "$EMULATOR_NAME" --device "$EMULATOR_DEVICE" --package "$EMULATOR_PACKAGE"
+
+#==========================
+# Install node & yarn berry
+#==========================
+
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash && \
+    apt-get -qqy install nodejs && npm install -g yarn && corepack enable && \
+    yarn set version 4.1.1
+
+
+# Install websokify and noVNC
+RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
+    python3 get-pip.py && \
+    pip3 install --no-cache-dir \
+        setuptools && \
+    pip3 install -U https://github.com/novnc/websockify/archive/refs/tags/v0.11.0.tar.gz
+
+RUN wget -O x11vnc.zip https://github.com/x11vnc/noVNC/archive/refs/heads/x11vnc.zip && \
```

Comment:
Is there a specific reason for this version rather than https://github.com/novnc/noVNC?  The x11vnc version looks like someone's unmaintained personal fork.

Reply:
Or even just `apt-get install novnc`